corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -107,8 +107,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
     var showMenu = function (menuResponse) {
         var menuListView = menusUtils.getMenuView(menuResponse);
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
-        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview;
-
+        var queryResponse = menuResponse.queryResponse;
+        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview && queryResponse && queryResponse.displays.length;
```

Comment:
so this may not work for all case types-- if `menuResponse.type === "query"` then there will be no `menuResponse.queryResponse` that's only for type entities. So if the response is type query then we would check for `menuResponse.collection.length` and for type entities it would be what you have

Reply:
I see what you mean. I'm pushing a change that I think addresses the two different menuResponse types.